python/servo/testing_commands.py

```diff
@@ -94,10 +94,12 @@ def run_test(self, prefix, args=[], release=False):
     @CommandArgument('--release', default=False, action="store_true",
                      help="Run with a release build of servo")
     @CommandArgument('--faster', default=False, action="store_true",
-                     help="Only check changed files and skip the WPT lint")
-    def test(self, params, render_mode=DEFAULT_RENDER_MODE, release=False, faster=False):
+                     help="Only check changed files and skip the WPT lint in tidy")
+    @CommandArgument('--no-progress', default=False, action="store_true",
+                     help="Don't show progress for tidy")
```

Comment:
I don't think we need this (or faster). `./mach test` is a special case, and we can choose appropriate defaults for it.